api/v1beta1/kafkacluster_types.go

```diff
@@ -127,7 +127,9 @@ type CruiseControlConfig struct {
 // CruiseControlTaskSpec specifies the configuration of the CC Tasks
 type CruiseControlTaskSpec struct {
 	// RetryDurationMinutes describes the amount of time the Operator waits for the task
-	RetryDurationMinutes int `json:"RetryDurationMinutes"`
+	RetryDurationMinutes int `json:"retryDurationMinutes"`
+	// DownScaleRetryDurationMinutes describes the amount of time the Operator waits during a downscale task
+	DownScaleRetryDurationMinutes int `json:"downScaleRetryDurationMinutes,omitempty"`
```

Comment:
do we really need two fields to specify the timeoutPeriod for upscale and downscale?